src/Core/NuGet/Core.NuGet-Beta/Package.nuspec

```diff
@@ -24,7 +24,7 @@
       </group>
       <group targetFramework="net46">
       </group>
-      <group targetFramework="dnxcore50">
+      <group targetFramework=".NETStandardApp,Version=v1.5">
         <dependency id="System.Collections" version="4.0.10-beta-22816" />
```

Comment:
As we shipping stable version of NuGet all references should be changed to the latest stable version of dependencies. No betas.